lib/actions/navigation/push_page.dart

```diff
@@ -0,0 +1,34 @@
+library push_page;
```

Comment:
there are just two actions now for all our navigation: PushPage and RemoveCurrentPage

Reply:
sweet